Gemfile

```diff
@@ -15,6 +15,8 @@ gem "rackup", "~> 2.2"
 gem "refinements", "~> 13.0"
 gem "sinatra-activerecord", "~> 2.0"
 gem "sqlite3", "~> 2.5"
+gem "dotenv", "~> 3.1"
```

Comment:
:sparkle: Thanks, I was too aggressive with this earlier and these need to be there.

:bulb: I might beat you to the punch as I've got some changes that would bring these back so you might not need this depending on if my work gets folded in before yours or not.